packages/app/src/runner/event-manager.ts

```diff
@@ -225,18 +225,6 @@ export class EventManager {
 
     this.reporterBus.on('runner:unpin:snapshot', this._unpinSnapshot.bind(this))
 
-    this.reporterBus.on('runner:resume', () => {
```

Comment:
Moved all the pause-related events to `runner/events/pausing.ts` to reduce some of the bloat in this file.